scripts/job-candidate/generate-tcUserId-for-existing-candidates.js

```diff
@@ -0,0 +1,190 @@
+const Sequelize = require('sequelize')
+const _ = require('lodash')
+const models = require('../../src/models')
+const helper = require('../../src/common/helper')
+const logger = require('../../src/common/logger')
+const config = require('config')
+
+// The number of job candidates to process in a single batch
+const JOB_CANDIDATES_BATCH_SIZE = Number(process.env.JOB_CANDIDATES_BATCH_SIZE) || 200
```

Comment:
How many total candidates are there, and how long will they take to process with a 1 second delay between batches?

Reply:
@testflyjets ,
We have 24311 job candidates records in Prouduction (8343 unique users - a single user can have multiple job candidates records)

To make the script run faster:
- Raised the batch size to 500
- Removed the wait time between batches
- Provided a static mapping file between the user handles which are present in TaaSDB and userIds for both Production and Dev, calls to member-api to get the member userId by handle will only be done for members who are not in the static file.

Details in this commit: https://github.com/topcoder-platform/taas-apis/pull/710/commits/3adc567a13a1acc346519352c1cdc230273ca5f3